Resources/Prototypes/SimpleStation14/Traits/disabilities.yml

```diff
@@ -20,7 +20,7 @@
   id: FakeMime
   name: Mute
   description: For one reason or another you will not speak, ever, and communicate through miming or writing.
-  cost: 6
+  cost: 4
```

Comment:
This seems very low, but I don't know the cost of positive traits off the top of my head. Are you sure being unable to speak in a social roleplaying game using primarily vocal long ranged communication is only 25% worse than not seeing colour?